programs/marginfi/src/instructions/marginfi_account/borrow.rs

```diff
@@ -105,8 +126,16 @@ pub fn lending_account_borrow<'info>(
             },
             bank: bank_loader.key(),
             mint: bank.mint,
-            amount: amount_pre_fee,
+            amount: amount_pre_fee + origination_fee_u64,
         });
+    } // release mutable borrow of bank
+
+    // The bank fee account gains the origination fee
+    {
+        let mut bank = bank_loader.load_mut()?;
+        let bank_fees_before: I80F48 = bank.collected_group_fees_outstanding.into();
+        let bank_fees_after: I80F48 = bank_fees_before.saturating_add(origination_fee);
```

Comment:
Lets skip this step if `origination_fee == 0`, I understand it has no effect, but I80F48 arithmetic increase compute usage.